content/derisking-government-tech/about.md

```diff
@@ -3,7 +3,33 @@ title: About
 layout: layouts/page
 tags: derisking-government-tech
+description: History and authors of the 18F De-risking Government Technology Guide 2.0.
 sidenav: false
 ---
 
-TK
+{% include 'derisking-government-tech/chapter_image.html' img_path: "assets/derisking-government-tech/img/icon.svg" %}
+
+The De-risking Government Technology Guide 2.0 consolidates content from the two parts of the original De-risking Guide: the [State Software Budgeting Handbook]({{ "https://guides.18f.gov/derisking/state-field-guide/" | url }}), released in August 2019, and the [Federal Agency Field Guide]({{ "https://guides.18f.gov/derisking/federal-field-guide/" | url }}), released in September 2020. In addition, the new section “[Working with a vendor development team]{{ "/derisking-government-tech/vendor-management/" | url }}” includes detailed guidance about vendor management. 
+
+These updates were made possible by the Office of Management and Budget’s [Facing a Financial Shock initiative]({{ "https://www.performance.gov/cx/life-experiences/facing-a-financial-shock/" | url }}). From May 2023 to June 2024, this funding enabled an 18F team to assess the feasibility of solutions to common challenges within state government IT acquisition, such as knowledge and skill gaps, focus on planning over results, and bureaucracy. After identifying vendor management to be a major area of concern among agency staff, the team took this opportunity to revise the De-risking Guide with relevant content.
+
+The current guide reflects the contributions of many current and former 18F staff:
+
+* Authors of the State Software Budgeting Handbook  
+  * Robin Carnahan, Randy Hart, and Waldo Jaquith  
+* Authors of the Federal Agency Field Guide  
+  * Mark Hopson, Victoria McFadden, Rebecca Refoy, and Alicia Rouault   
+* Contributors to the Federal Agency Field Guide  
+  * Alan Atlas, Heather Battaglia, T. Carter Baxter, Kelsey Foley, Waldo Jaquith, Ryan Johnson, Brandon Kirby, Ian Lee, Miatta Myers, Steven Reilly, Stephanie Rivera, Peter Rowland, and Greg Walker  
+* Authors and editors of the De-risking Government Technology Guide 2.0  
+  * Alan Atlas, Elizabeth Ayer, Brian Burns, Stacy Dion, Randy Hart, Mark Hopson, Selena Juneau-Vogel, Miatta Myers, Laura Poncé, Peter Rowland, Amelia Wong, and Lindsay Young
+
+With thanks to the many people who shared feedback on the guide’s content:
+
+* From 18F: Claire Blaustein, Lalitha Jonnalagadda, Amanda Kennedy, Jason Nakai, Allison Norman, Cale Rubenstein   
+* From Technology Transformation Services: Davida Marion   
+* The several state employees who participated in user research
+
+Special thanks to Igor Korenfeld for designing the PDF template of the guide; Mel Choyce for designing new visual elements, updating the layout, and refining the PDF template; Nate Borrebach for building the new online version; and Komal Rasheed, for her invaluable guidance and support throughout the entirety of the Facing Financial Shock project.
```

Comment:
suggestion (non-blocking): This is long enough that a list might be appropriate

Reply:
Thanks, I'll follow up on the content design here.

Reply:
Per @ameliaswong I've updated this to a bulleted list. Thanks!